dev/io.openliberty.data.internal_fat_nosql/bnd.bnd

```diff
@@ -33,4 +33,4 @@ fat.project: true
 	io.openliberty.jakarta.servlet.6.0,\
 	io.openliberty.jakarta.transaction.2.0,\
 	io.openliberty.org.eclipse.microprofile.config.3.0,\
-	io.openliberty.org.eclipse.jnosql.1.0
```

Comment:
You'll want to add the following line to this file:

fat.test.container.images: mongo:6.0.6